src/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(n) {
+  background: url(./images/star-active.svg) no-repeat center;
```

Comment:
you should change only background-image, other background-properties you leave the same, so you don't repeat them here (no-repeat, center)